src/index.html

```diff
@@ -1,13 +1,50 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="img"
+        src="./images/imac.jpeg"
+        alt="#"
+      >
+
+      <div class="title">
+        <h3 class="title-title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+      </div>
+
+      <div class="phrases">
+        <p class="phrase phrase--position_right">Код товара:195434</p>
+      </div>
+
+      <div  class = "stars">
```

Comment:
```suggestion
      <div class="stars stars--4">
```
Не надо оставлять пробелы между class =  '
И у этого блока еще должен быть класс-модификтор, который отвечает за количество активных звезд